docusaurus/docs/how-to-guides/data-source-plugins/config-test.md

```diff
@@ -0,0 +1,135 @@
+---
+id: data-source-configuration
+title: Configure data sources
+description: A guide to data source configuration.
+keywords:
+  - grafana
+  - data source plugin
+  - datasource
+  - data source plugin configuration
+---
+
+# Best practices for data source plugin configuration
+
+To deliver a consistent and user-friendly experience to Grafana users, we recommend that Grafana plugin developers follow uniform guidelines when building data source plugins. This guide outlines recommended best practices to help you achieve this goal.
+
+## Logos
+
+If your plugin requires a logo, follow these recommendations:
+
+- Ensure you obtain permission from the trademark holder.
+- Follow the brand guidelines, typically available in a company’s media kit or press kit.
+- Use logos with a transparent background, avoiding white backgrounds.
+- Logos should be in SVG format.
+
+**Example:** MongoDB  
+**Example:** Datadog
+
+## Description and required fields
+
+We recommend using the `DataSourceDescription` component from the [@grafana/experimental](https://github.com/grafana/grafana-experimental) repository. The plugin name and documentation link are required, but details about other fields is optional.
```

Comment:
I am against suggesting people from using any kind of component from the grafana-experimental package. It is called experimental because we will break things there and we can't guarantee this document will keep up to date with changes in that repository.

If we are going to start recommending that component we will have to promote that component to grafana-ui.

@sympatheticmoose thoughts?

Reply:
Let me know if we need to remove all references to components from experimental.

Reply:
Agree with Esteban, experimental should not be recommended

Reply:
The same component lives in `@grafana/plugin-ui`. So maybe reference that one the same way instead? See this [link](https://github.com/grafana/plugin-ui/blob/main/src/components/ConfigEditor/DataSourceDescription.tsx) for more details.

Reply:
done. removed reference to experimental, replaced with plugin-ui link